Dockerfile-api

```diff
@@ -3,33 +3,16 @@
 # Defines the runtime for the arXiv search API, which provides a metadata
 # query API backed by Elasticsearch.
 
-FROM arxiv/base:0.14.3
+FROM arxiv/base:0.16.6
 
 WORKDIR /opt/arxiv
 
-# Install MySQL.
-RUN yum install -y which mysql mysql-devel
```

Comment:
I think that the mysql client drivers are intended to be supplied by the base image:
https://github.com/arXiv/arxiv-base/blob/09643fbcafab3bda9dda55656c41574d4f494afb/Dockerfile#L34

If mysql client and server are needed for this docker image please ignore this comment.